openapi-contracts/src/test/resources/domains/conversation/v1/messages/request/SendTextMessageRequestDto.json

```diff
@@ -0,0 +1,76 @@
+{
+  "app_id": "my app ID",
+  "recipient": {
+    "contact_id": "my contact ID"
+  },
+  "message": {
+    "text_message": {
+      "text": "This is a text message."
+    },
+    "explicit_channel_message": {
+      "KAKAOTALK": "foo value"
+    },
+    "explicit_channel_omni_message": {
+      "MESSENGER": {
+        "message": {
```

Comment:
To be checked for this property. I think card_message should be one level up

Reply:
_(noted for specs question)_